dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -73,6 +73,7 @@ h1 {
 }
 
 h2 {
+  margin: 25px 0 10px 0;
```

Comment:
Definitely a risk here, but we need to be able to modify these base styles in order to align with Pegasus.  Hopefully any component that doesn't want the "default" site-wide margin on `<h2>` has already set the correct margins.  If not I'll have to fix any issues as they arise 😄.